<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotWidget</class>
 <widget class="QWidget" name="ScreenshotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>777</width>
    <height>479</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>777</width>
    <height>479</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="screen_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>开始截图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>退出截图</string>
   </property>
  </widget>
  <widget class="QLabel" name="init_img">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>511</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mini_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>最小化到托盘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>选择截图保存路径</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
